netebpfext/sys/net_ebpf_ext_drv.c

```diff
@@ -148,13 +148,10 @@ DriverEntry(_In_ DRIVER_OBJECT* driver_object, _In_ UNICODE_STRING* registry_pat
 {
     NTSTATUS status;
 
+    NET_EBPF_EXT_LOG_ENTRY();
```

Comment:
my understanding is that NET_EBPF_EXT_LOG_ENTRY() should always be paired with a similar exit call like NET_EBPF_EXT_LOG_EXIT()

Reply:
fixed